src/test/kotlin/no/nav/syfo/persistering/PersisterSykmeldingQueriesKtTest.kt

```diff
@@ -0,0 +1,38 @@
+package no.nav.syfo.persistering
+
+import java.util.UUID
+import no.nav.syfo.testutil.TestDB
+import no.nav.syfo.testutil.dropData
+import no.nav.syfo.testutil.getSykmeldingsopplysninger
+import no.nav.syfo.testutil.testSykmeldingsdokument
+import no.nav.syfo.testutil.testSykmeldingsopplysninger
+import org.amshove.kluent.shouldEqual
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+internal class PersisterSykmeldingQueriesKtTest : Spek({
```

Comment:
Blir denne kjørt av spek når den er internal?

Reply:
Jeg trodde det, men jeg fjerner internal for sikkerhets skyld. Av en eller annen grunn lager alltid IntelliJ internal-klasser for meg når jeg ber den om å generere tester, så glemmer jeg bare å fjerne taggen.